packages/commons/src/fileManager.ts

```diff
@@ -32,7 +31,20 @@ const mockFileManager: FileManager = {
   },
 };
 
-function initFileManager(): FileManager {
+export function initFileManager(
+  config:
+    | {
+        mockFileManager: true;
+      }
+    | {
+        mockFileManager: false;
+        s3AccessKeyId: string;
+        s3SecretAccessKey: string;
+        s3Region: string;
+        s3BucketName: string;
+        eserviceDocsPath: string;
+      }
```

Comment:
is it the same as this?
```suggestion
  config: FileManagerConfig
```

Reply:
Yes, @Viktor-K in his PR https://github.com/pagopa/interop-be-monorepo/pull/82 refactor the "config system" so this fix is already included there.